Marker/Classes/Marker.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Marker.swift
+//  Marker
+//
+//  Created by Htin Linn on 5/15/17.
+//  Copyright © 2017 Prolific Interactive. All rights reserved.
+//
+
+import Foundation
+
+/// Returns formatted Markdown text as an attributed string.
+///
+/// - Parameters:
+///   - markdownText: String with Markdown tags.
+///   - textStyle: Text style object containing style information.
+/// - Returns: Formatted Markdown text.
+public func attributedMarkdownString(from markdownText: String,
```

Comment:
Why not having a static function inside of a `Marker` type then? I rarely use global functions, and for this specific scenario I don't know if it's needed. Also I believe the autocompletion will be better if you have a `Marker` type, I remember that for Alamofire you don't have autocomplete until you start typing the `request` function